partner_create_by_vat/__openerp__.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# ©  2015 Forest and Biomass Services Romania
+# See README.rst file on addons root folder for license details
+
+{
+    'name': 'Partner Create by VAT',
+    'summary': 'Create Partners by VAT from VIES Webservice',
+    'version': '8.0.1.0.0',
+    'category': 'Customer Relationship Management',
+    'author': 'Forest and Biomass Services Romania, '
+              'Odoo Community Association (OCA)',
+    'website': 'https://www.forbiom.eu',
+    'license': 'AGPL-3',
+    'application': False,
+    'installable': True,
+    'external_dependencies': {
+        'python': ['stdnum', 'suds'],
```

Comment:
suds is required by stdnum, it i only leaved stdnum...i had error about the suds package not installed...